### PR TITLE
Create cloud-download-manager.yml

### PR DESCRIPTION
**Pull request by besoeasy**
Cloud Download Manager

A cloud download manager is a software application that manages downloads from cloud storage services. Cloud download managers typically allow users to upload files to cloud storage, and then manage the download process of those files from the cloud to their local devices.

### CODE REVIEW

File: _data/projects/cloud-download-manager.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+site: https://github.com/besoeasy/Cloud-Download-Manager
+tags:
+- torrent
+- cloud
+- aria2
+- telegram
+- downloader
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/besoeasy/Cloud-Download-Manager/labels/up-for-grabs
```
Comment (shiftkey) [inline _data/projects/cloud-download-manager.yml]: 🤔 this label doesn't exist in the project, and it's not clear to me which tasks in the repository would belong to this label. Are you able to craft some tasks and ensure this link points to the list of tasks?

Comment (besoeasy) [reply _data/projects/cloud-download-manager.yml]: upforgrabs:
  name: enhancement
  link: https://github.com/besoeasy/Cloud-Download-Manager/labels/enhancement

**Comment by besoeasy** [comment]
Thanks for taking time, i have added the correct labels !

@shiftkey

File: _data/projects/cloud-download-manager.yml (hunk 1 above)
Comment (shiftkey) [inline _data/projects/cloud-download-manager.yml]:
```suggestion
  name: enhancement
  link: https://github.com/besoeasy/Cloud-Download-Manager/labels/enhancement
```

File: _data/projects/cloud-download-manager.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+- aria2
+- telegram
+- downloader
+upforgrabs:
+  name: enhancement
+  link: https://github.com/besoeasy/Cloud-Download-Manager/labels/enhancement
+stats:
+  issue-count: 0
+  last-updated: '2023-04-12T05:22:30Z'
+  fork-count: 0
```
Comment (shiftkey) [inline _data/projects/cloud-download-manager.yml]: :art: dropping this now to get around the CI issue

```suggestion
```

**Pull request merged**